doc/source/user_guide/indexing.rst

```diff
@@ -573,7 +573,7 @@ without using a temporary variable.
 
 .. _indexing.deprecate_ix:
 
-IX indexer is deprecated
+ix indexer is deprecated
```

Comment:
I changed this to 'ix' as it is a deprecated function referred to as such, not 'IX'.  Please let me know if there's an issue with this @dsaxton

Reply:
ix is _removed_, this should be taken out entirely  (doesnt have to be this PR)